app/views/organizations/list_requests.html.erb

```diff
@@ -21,7 +21,7 @@
       <td><%= organization.created_at %></td>
       <td><%= link_to 'Show information', organization %></td>
       <td><%= link_to 'Accept', accept_organization_path(organization), method: :post, class: 'btn btn-success' %></td>
-      <td><%= link_to 'Reject', reject_reason_input_organization_path(organization), method: :post, class: 'btn btn-danger' %></td>
+      <td><%= link_to 'Reject', reject_reason_input_organization_path(organization), class: 'btn btn-danger' %></td>
```

Comment:
Not a big fan of allowing changing state (i.e allowing to reject/accept org with a get request)